.ci-integration-tests-iso4.yml

```diff
@@ -1,9 +1,11 @@
 pipeline_template: ci/inmanta-tools/Jenkinsfile-integration-tests
 repo_name: pytest-inmanta-lsm
 lock:
-    label: iso3-test
+    label: pytest-inmanta-lsm-iso4-dev
     variable: INMANTA_LSM_HOST
     quantity: 1
+additional_constraints:
+    - inmanta-lsm~=1.6
```

Comment:
Why `inmanta-lsm~=1.6`? The `setup.py` file on `inmanta-service-orchestrator` repository mentions `inmanta-lsm~=1.5.dev` for the `iso4` branch.

Reply:
Why don't we put this constraint into the `setup.py` file?

Reply:
> Why don't we put this constraint into the `setup.py` file?

I see. The only active version is present on the master branch.

Reply:
You're right that I should add the `.dev`. I made it `1.6` instead of `1.5` because of the issue with 1.5 I mentioned on slack yesterday.